components/candy-machine/CreateCandyMachine.tsx

```diff
@@ -40,37 +40,37 @@ const CreateCandyMachine: FC = () => {
 
     function isFormValid(): boolean {
         if (files.length === 0) {
-            setErrorMessage('There are no files to upload')
+            setError('There are no files to upload')
```

Comment:
I think we could get rid of the `error and setError useState` completely and call the `addNotification` function on each case.
Maybe even invoking a new function that takes the error string and populates the new state of the notification.

```ts
function populateNotificationError(newError: string){
   addNotification({
      message: 'There was an error creating the candy machine \n' + newError,
      type: NotificationType.Error,
   })
}
```

Even more, since this approach is used in `CreateCandyMachine.tsx and UpdateCandyMachine.tsx` this method could be added to the useNotifications hook and returned, so on these 2 files we can use this method.